homeworks/magurmarian/hw3/testbed.py

```diff
@@ -0,0 +1,28 @@
+from pyats.utils.fileutils import FileUtils
+from pyats.aetest import Testcase, test
+
+
+class Smoke(Testcase):
+    @test
+    def copy_to_local_pc(self, tb):
+        with FileUtils(testbed=tb) as futils:
+            futils.copyfile(
+                source='scp://192.168.195.143/home/adminaccount/some.txt',
+                destination='/home/marian/github/pyatsTraining/homeworks/magurmarian/hw3')
+
+
+class Smoke1(Testcase):
+    @test
+    def copy_to_remote_pc(self, tb):
+        with FileUtils(testbed=tb) as futils:
+            futils.copyfile(
+                source='/home/marian/github/pyatsTraining/homeworks/magurmarian/hw3/some.txt',
+                destination='scp://192.168.195.143/home/adminaccount')
+
+
+class Smoke2(Testcase):
```

Comment:
@marianmagur  Is there really a point to split these 3 tests into different classes, considering you basically gave them the same name?
Also "Smoke2/check_existence_file" could be implemented alongside "Smoke1/copy_to_remote_pc", since it's this test's logical extension.